pymedphys/_dicom/anonymisation_strategy.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2020 Stuart Swerdloff, Matthew Jennings, Simon Biggs
```

Comment:
Let's change this here to:

```python
# Copyright (C) 2020 Stuart Swerdloff
# Copyright (C) 2018 Matthew Jennings, Simon Biggs
```

To indicate that it is borrowing from work done in 2018, but the 2020 updates were yours.

Reply:
next push, as soon as I can see what is still failing